app/models/concerns/common_metadata.rb

```diff
@@ -19,6 +19,9 @@ module CommonMetadata
     property :state, predicate: ::F3Access.objState, multiple: false do |index|
       index.as :stored_searchable, :facetable
     end
+    property :workflow_note, predicate: ::RDF::Vocab::MADS.note do |index|
```

Comment:
I thought this was going to be ::RDF::Vocab::MODS.note.  MADS.note and MODS.note seem pretty equivalent, but MADS is typically used for authorities records, and MODS is typically used for resources.